Fix for #18176

bendk opened the pull request:
We weren't including widgetsapi in setup.py because it was only used by
extensions.  Since extensions are dynamically loaded, this messed up the
module finder.  Manually import the module to fix the issue.

paulswartz commented:
I think that the fix for this should be on the py2exe end, rather than the importing end.  That file now throws up a pyflakes error, since it's not used in the file.

Commit:
Fix for #18176

Fixed setup.py so that py2exe knows about all the packages we give to
distutils.  This makes it include the entire miro.frontend.widgets package,
and in particular makse it include the widgetsapi module.  Before, widgetsapi
wasn't being found by the module finder since it's only used by extensions
which are dynamically loaded.

bendk commented:
Definitely a good call paul.  When I was looking it over, I realized that it could be fixed quite cleanly by improving setup.py.

paulswartz commented:
Much better!
